modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ButtonDropDown/index.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+import ClayDropDown, {Align} from '@clayui/drop-down';
+import ClayIcon from '@clayui/icon';
+
+const ButtonDropDown = ({
+	label,
+	align = Align.BottomRight,
+	active,
+	setActive,
+	items,
+	...props
+}) => {
+	return (
+		<ClayDropDown
+			active={active}
+			alignmentPosition={align}
+			onActiveChange={setActive}
+			trigger={
+				<button className="btn btn-primary px-3 py-2">
+					{label}
+
+					<ClayIcon className="ml-2" symbol="caret-bottom" />
+				</button>
```

Comment:
Should be good use our Button, present in common/components ou the Botton provides by Clay (@clay/core)

Reply:
Changing to Button present in common/components it's not working, doesn't show drop down items

Reply:
It's working with the Button provides by Clay (@clay/core)
Thank you!